doc/source/skips/0-skip-process.rst

```diff
@@ -0,0 +1,300 @@
+.. _skip0:
+
+============================
+SKIP 0 — Purpose and Process
+============================
+
+:Author: Jarrod Millman <millman@berkeley.edu>
+:Author: Juan Nunez-Iglesias <juan.nunez-iglesias@monash.edu>
+:Status: Active
+:Type: Process
+:Created: 2019-07-30
+
+
+What is a SKIP?
+--------------
+
+SKIP stands for SciKit-Image Proposal. A SKIP is a design document providing
+information to the NumPy community, or describing a new feature for
+scikit-image or its processes or environment. The SKIP should provide a
+concise technical specification of the feature and a rationale for the feature.
+
+We intend SKIPs to be the primary mechanisms for proposing major new
+features, for collecting community input on an issue, and for
+documenting the design decisions that have gone into NumPy. The SKIP
+author is responsible for building consensus within the community and
+documenting dissenting opinions.
+
+Because the SKIPs are maintained as text files in a versioned
+repository, their revision history is the historical record of the
+feature proposal [1]_.
+
+
+Types
+^^^^^
+
+There are three kinds of SKIPs:
+
+1. A **Standards Track** SKIP describes a new feature or implementation
+   for scikit-image.
+
+2. An **Informational** SKIP describes a scikit-image design issue, or provides
+   general guidelines or information to the Python community, but does not
+   propose a new feature. Informational SKIPs do not necessarily represent a
+   scikit-image community consensus or recommendation, so users and
+   implementers are free to ignore Informational SKIPs or follow their advice.
+
+3. A **Process** SKIP describes a process surrounding scikit-image, or
+   proposes a change to (or an event in) a process. Process SKIPs are
+   like Standards Track SKIPs but apply to areas other than the NumPy
```

Comment:
NumPy language—guess even in NumPy we forgot to update this properly from the Python version